libcontainer/configs/validate/rootless.go

```diff
@@ -55,6 +55,9 @@ func rootlessEUIDMount(config *configs.Config) error {
 	for _, mount := range config.Mounts {
 		// Check that the options list doesn't contain any uid= or gid= entries
 		// that don't resolve to root.
+		if !strings.Contains(mount.Data, "id=") {
```

Comment:
I think the current comment on top of this is not very clear. That explains what the for loop does, so I think that comment outside of the loop seems better. And here I'd explain that we are checking for "id" as a way to see if it containers uid or gid and fail early if it doesn't.

Reply:
Fixed, thanks